tests/test_dataapi.py

```diff
@@ -0,0 +1,100 @@
+import unittest
+import responses
+from learnosity_sdk.request import DataApi
+
+security = {
+    'consumer_key': 'MY_CONSUMER_KEY',
+    'domain': 'localhost'
+}
+consumer_secret = 'MY_CONSUMER_SECRET'
+request = {
+    'references': ['1', '2'],
```

Comment:
To be very precise, this request should also ask for `"limit": 1`

Reply:
The response is mocked anyway. I just mocked it so that the server appears to only have one item.

Reply:
Same as above, as the mocked response behave as if we had a limit of 1, which is not the default, we should set it there.

For extra credit, we should actually request items that are auto-provisioned to enable easier  writing of integration tests later on.